tracer/src/Datadog.Trace/RemoteConfigurationManagement/RcmSubscriptionManager.cs

```diff
@@ -208,6 +214,32 @@ private GetRcmRequest BuildRequest(RcmClientTracer rcmTracer, string? lastPollEr
         return rcmRequest;
     }
 
+    private async Task<GetRcmResponse?> TrySendRequest(RcmClientTracer rcmClientTracer, Func<GetRcmRequest, Task<GetRcmResponse?>> func)
+    {
+        try
+        {
+            var request = BuildRequest(rcmClientTracer, _lastPollError);
+            _lastPollError = null;
+
+            return await func(request).ConfigureAwait(false);
+        }
+        catch (RemoteConfigurationDeserializationException e)
+        {
+            Log.Error(e, "Error sending request to RCM endpoint: serialization error");
+            // serialization errors should be reported to the agent
+            // but we want the _inner_ message
+            _lastPollError = $"{e.Message}: {e.InnerException?.Message}";
```

Comment:
`e.Message` is controlled by the tracer while the inner exception is not and could be anything?

Reply:
Well, technically, yes, but it would specifically be a deserialization error, which would indicate a bug to address

Reply:
then as a nitpick... Is it going to be locale independent?

Reply:
It depends... exception messages from the _runtime_ are normally localized, but as this is a serialization error (which is using a vendored library currently), then they likely _won't_ be